doc/nrf/working_with_nrf/nrf53/thingy53.rst

```diff
@@ -243,25 +243,29 @@ See :file:`thingy53_nrf5340_cpuapp.conf` in a compatible sample for how this is
 USB
 ===
 
-An application compatible with the Nordic Thingy:53 is expected to enable USB CDC ACM as backend for logging.
```

Comment:
Consider removing information about applications/samples that are compatible with Thingy:53 in the NCS. Currently all of the applications and samples are built together with MCUboot so that basic compatibility with the preprogrammed bootloader and default logging method is added out-of-the box (and OTA DFU can be enabled for any sample using a single Kconfig option).

Reply:
Also multi-image build system will be always used (application is in general built together with MCUboot bootloader and netcore image.

Reply:
I am also not sure if we should explicitly mention building separate image for the network core (I think such image would be misaligned with Thingy:53 support in the NCS in current version). We could simply inform that we in general build for app core and net core image is automatically added as a child image. I think

See:
`The build target should be thingy53_nrf5340_cpuapp or thingy53_nrf5340_cpuapp_ns when building samples for the application core and thingy53_nrf5340_cpunet when building for the network core.` in the current documentation

Reply:
You could also update the `Firmware update using external debug probe` section. Currently all of the samples/apps are compatible with preprogrammed MCUboot bootloader.

Reply:
At the beginning of `Thingy:53 application guide` you could mention that in general samples and applications are compatible with preprogrammed bootloader out-of-the box and only SMP OTA DFU needs to be explicitly enabled using an additional Kconfig option (most of the configuration is done in the background and user does not need to actually do anything there).